### PR TITLE
Mike harley

### PR DESCRIPTION
**Pull request by mharley12345**
desktop 1024 px was fairly easy got stuck on the table and mobile some parts I got others not so much. Started to beat head against wall had to walk away.

### CODE REVIEW

**Comment by mharley12345** [comment]
Just pushed cleaned up version of css and html. Used better semantic markup and removed css for services and about page  that I copied and pasted by accident.

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,390 @@
...
+}
+body {
+	max-width: 1200px;
+	margin: 0 auto;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+header {
+	display: flex;
+	justify-content: flex-start;
```
Comment (adammathieson) [inline css/index.css:73]:
```suggestion
```

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,390 @@
...
+	flex-wrap: nowrap;
```
Comment (adammathieson) [inline css/index.css:74]:
```suggestion
```

File: css/index.css [hunk 3]
```diff
@@ -1,3 +1,390 @@
...
+	align-items: flex-start;
```
Comment (adammathieson) [inline css/index.css:75]:
```suggestion
```

Comment (adammathieson) [inline css/index.css:75]:
```suggestion
	align-items: center;
```

Comment (adammathieson) [reply css/index.css:75]: aligns the content centered on cross axis

File: css/index.css (hunk 1 above)
Comment (adammathieson) [inline css/index.css:73]:
```suggestion
```

File: css/index.css [hunk 4]
```diff
@@ -1,3 +1,390 @@
...
+
+
+
+
+}
+header hr {
+	display: none;
+}
+nav {
+	width: 90%;
```
Comment (adammathieson) [inline css/index.css:87]:
```suggestion
	width: 80%;
```

Comment (adammathieson) [reply css/index.css:87]: 80% will give more room for the logo (and some padding)

File: css/index.css [hunk 5]
```diff
@@ -1,3 +1,390 @@
...
+	display:flex;
+	justify-content: space-around;
```
Comment (adammathieson) [inline css/index.css:89]:
```suggestion
	justify-content: space-between;
```

Comment (adammathieson) [reply css/index.css:89]: spreads the end <a> tags to the margin (like the design file)

File: css/index.css [hunk 6]
```diff
@@ -1,3 +1,390 @@
...
+}
+nav a{
+	text-decoration: none;
+	color: black;
+}
+
+.top-sec {
+
+	 display:flex;
+	 justify-content: space-around;
```
Comment (adammathieson) [inline css/index.css:101]:
```suggestion
	 justify-content: space-evenly;
```

File: css/index.css [hunk 7]
```diff
@@ -1,3 +1,390 @@
...
+	 flex-wrap: wrap;
```
Comment (adammathieson) [inline css/index.css:102]:
```suggestion
```suggestion

```
```

File: css/index.css [hunk 8]
```diff
@@ -1,3 +1,390 @@
...
+
```
Comment (adammathieson) [inline css/index.css:103]:
```suggestion
align-items: center;
```

Comment (adammathieson) [reply css/index.css:103]: should take care of distortion to the image

File: css/index.css [hunk 9]
```diff
@@ -1,3 +1,390 @@
...
+
+}
+
+.top-sec img {
```
Comment (adammathieson) [inline css/index.css:107]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:107]: This tag is not needed with the other changes

File: css/index.css [hunk 10]
```diff
@@ -1,3 +1,390 @@
...
+
+	display: flex;
```
Comment (adammathieson) [inline css/index.css:109]:
```suggestion

```

File: css/index.css [hunk 11]
```diff
@@ -1,3 +1,390 @@
...
+	padding-top: 45px;
```
Comment (adammathieson) [inline css/index.css:110]:
```suggestion
	
```

File: css/index.css [hunk 12]
```diff
@@ -1,3 +1,390 @@
...
+	width: 45%;
```
Comment (adammathieson) [inline css/index.css:111]:
```suggestion

```

File: css/index.css [hunk 13]
```diff
@@ -1,3 +1,390 @@
...
+	height: 45%;
```
Comment (adammathieson) [inline css/index.css:112]:
```suggestion

```

File: css/index.css [hunk 14]
```diff
@@ -1,3 +1,390 @@
...
+.bottom-img {
+
+	 display:flex;
+	 justify-content: space-around;
+	 width: 100%;
+
+}
+.bottom-img img {
+
+width: 100%;
```
Comment (adammathieson) [inline css/index.css:180]:
```suggestion
max-width: 100%;
```

Comment (adammathieson) [reply css/index.css:180]: use max-width for better scaling

File: css/index.css [hunk 15]
```diff
@@ -1,3 +1,390 @@
...
+	padding-left: 10px;
+}
+}
+	
+
+@media (max-width: 800px) {
+
+	body {
+		max-width: 800px;
+		max-height: 1370px;
```
Comment (adammathieson) [inline css/index.css:272]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:272]: Typically you won't need to set a height, most styling methods will result in the height being set by what space the content needs to take up

File: css/index.css [hunk 16]
```diff
@@ -1,3 +1,390 @@
...
+		margin: 20px;
+		resize: both;
```
Comment (adammathieson) [inline css/index.css:274]:
```suggestion
	
```

Comment (adammathieson) [reply css/index.css:274]: not needed

File: css/index.css [hunk 17]
```diff
@@ -1,3 +1,390 @@
...
+	
+	}
+	header{
+	display: flex;
+	flex-wrap:nowrap;
```
Comment (adammathieson) [inline css/index.css:279]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:279]: not needed

File: css/index.css [hunk 18]
```diff
@@ -1,3 +1,390 @@
...
+		background: black;
+	}
+
+
+}
+@media (max-width: 500px) {
+    
+	body {
+		max-width: 500px;
+		max-height: 2047px;
```
Comment (adammathieson) [inline css/index.css:343]:
```suggestion
	
```

Comment (adammathieson) [reply css/index.css:343]: not needed

File: css/index.css [hunk 19]
```diff
@@ -1,3 +1,390 @@
...
+		resize: both;
```
Comment (adammathieson) [inline css/index.css:344]:
```suggestion
	
```

Comment (adammathieson) [reply css/index.css:344]: not needed

File: css/index.css [hunk 20]
```diff
@@ -1,3 +1,390 @@
...
+	
+	}
+	header{
+		display: flex;
```
Comment (adammathieson) [inline css/index.css:348]:
```suggestion
```

Comment (adammathieson) [reply css/index.css:348]: property is already set at the top of the file

File: css/index.css [hunk 21]
```diff
@@ -1,3 +1,390 @@
...
+		flex-wrap:nowrap;
```
Comment (adammathieson) [inline css/index.css:349]:
```suggestion
		
```

Comment (adammathieson) [reply css/index.css:349]: not needed

File: css/index.css [hunk 22]
```diff
@@ -1,3 +1,390 @@
...
+		flex-direction: column-reverse;
+		}
+	
+header img {
+	align-self: center;
```
Comment (adammathieson) [inline css/index.css:354]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:354]: not needed

File: css/index.css [hunk 23]
```diff
@@ -1,3 +1,390 @@
...
+	padding-left: 20px;
```
Comment (adammathieson) [inline css/index.css:355]:
```suggestion
	
```

Comment (adammathieson) [reply css/index.css:355]: not needed

File: css/index.css [hunk 24]
```diff
@@ -1,3 +1,390 @@
...
+}
+.navigation-main {
+
+	display: flex;
```
Comment (adammathieson) [inline css/index.css:359]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:359]: property is already set

File: css/index.css [hunk 25]
```diff
@@ -1,3 +1,390 @@
...
+	flex-direction: column;
+	flex-wrap: nowrap;
```
Comment (adammathieson) [inline css/index.css:361]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:361]: not needed

File: css/index.css [hunk 26]
```diff
@@ -1,3 +1,390 @@
...
+}
+  a {
+	display: flex;
```
Comment (adammathieson) [inline css/index.css:364]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:364]: property cannot be applied to this element

File: css/index.css [hunk 27]
```diff
@@ -1,3 +1,390 @@
...
+	flex-direction: row;
```
Comment (adammathieson) [inline css/index.css:365]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:365]: not applied to this element- would go on parent (already applied by default)

File: css/index.css [hunk 28]
```diff
@@ -1,3 +1,390 @@
...
+	flex-wrap: nowrap;
```
Comment (adammathieson) [inline css/index.css:366]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:366]: not needed

File: css/index.css [hunk 29]
```diff
@@ -1,3 +1,390 @@
...
+	align-self: stretch;
```
Comment (adammathieson) [inline css/index.css:367]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:367]: not needed

File: css/index.css [hunk 30]
```diff
@@ -1,3 +1,390 @@
...
+	align-items: center;
```
Comment (adammathieson) [inline css/index.css:368]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:368]: not needed

File: css/index.css [hunk 31]
```diff
@@ -1,3 +1,390 @@
...
+	padding: 20px;
+	padding: left 50px;
```
Comment (adammathieson) [inline css/index.css:370]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:370]: Should start to use less of padding/margin for major positioning; fine for details and minor adjustments

File: css/index.css [hunk 32]
```diff
@@ -1,3 +1,390 @@
...
+	border-top: 1px solid lightgray;
+	border-bottom: 1px solid lightgray;
```
Comment (adammathieson) [inline css/index.css:372]:
```suggestion

```

Comment (adammathieson) [reply css/index.css:372]: applying both of these properties will result in a doubling effect of the line

File: css/index.css [hunk 33]
```diff
@@ -1,3 +1,390 @@
...
+	
+	}
+
+
```
Comment (adammathieson) [inline css/index.css:376]:
```suggestion
a:last-child a {
border-bottom: 1px solid lightgray;
```

Comment (adammathieson) [reply css/index.css:376]: adds border to last child of type <a>

**Review by adammathieson — COMMENTED** [review]
The project is coming along nicely; while I made significant changes, really I mostly removed properties and stying attributes that either weren't doing anything, or in some cases were actually limiting your ability to positions certain elements. This is typical when getting a feel for styling, there's a feeling of not wanting to undue work, but never underestimate the liberation that comes from tearing up the paper and starting fresh. Look over the changes and see if you can see how in some case less is in fact more. Good work overall.

**Pull request merged**